build.sbt

```diff
@@ -83,7 +83,7 @@ libraryDependencies ++= Seq(
   "com.gu" %% "content-api-client-aws" % "0.7.6",
   "com.gu" %% "content-api-client-default" % capiClientVersion,
   "com.gu" %% "editorial-permissions-client" % "3.0.0",
-  "com.gu" %% "fapi-client-play30" % "18.0.0",
+  "com.gu" %% "fapi-client-play30" % "19.0.0-PREVIEW.fpallow-media-atoms.2025-05-20T0845.d382d883",
```

Comment:
Needs to be updated when https://github.com/guardian/facia-scala-client/pull/355 is merged.